src/allmydata/scripts/common.py

```diff
@@ -2,9 +2,14 @@
 
 import os, sys, urllib, textwrap
 import codecs
-from six.moves.configparser import NoSectionError
 from os.path import join
+
+# BBB: Python 2 compatibility
+from builtins import str
```

Comment:
Maciej had a good point that we should probably use `from future.builtins import str` just for clarity on what's going on.

Reply:
Thanks for the reminder.  I used the format above because it's what's "recommended" in [the python-future docs](https://python-future.org/compatible_idioms.html#unicode) but I'd missed [the place in the wiki where we're deviating from that](https://tahoe-lafs.org/trac/tahoe-lafs/wiki/Python3#PortingaspecificPythonfile). I'll make that change now.